layouts/partials/outputformat.html

```diff
@@ -0,0 +1,12 @@
+{{/* Return the current page outputformat */}}
+{{ $alts := newScratch }}
+{{ $format := "unknown" }}
+{{ range .AlternativeOutputFormats }}
+    {{ $alts.Set .Name true }}
```

Comment:
Does this logic require that there are exactly two formats (a main one and a single alternative)? If so, that seems fragile.

Reply:
This file is an exact copy from the theme?
Do you need this file?